README.md

```diff
@@ -81,41 +47,37 @@ Below is an example of sidecar container which needs to be added to the CSI driv
       initialDelaySeconds: 10
       timeoutSeconds: 3
       periodSeconds: 2
-      failureThreshold: 1
-   volumeMounts:
-    - mountPath: /csi
```

Comment:
Should we keep this mountPath (and add volume definition), since this is needed for sidecar to plugin communication?

Reply:
`volumeMounts` for the CSI socket restored (I left out the other volumes)